lib/generators/templates/wor_paginate.rb

```diff
@@ -0,0 +1,11 @@
+Wor::Paginate.configure do |config|
+  config.default_per_page = 25
+  # config.default_page = 1
+
+  config.page_param = :page
+  config.per_page_param = :per
+
+  # In case you want to use other format for your response, you can override our formatter here
+  # Check https://url-to-the-docs
```

Comment:
Not now, i'll add a link to Wor::Paginate::Formatter for now & also an issue for this